tests/wpt/metadata/webdriver/tests/take_element_screenshot/screenshot.py.ini

```diff
@@ -1,2 +1,7 @@
 [screenshot.py]
-  disabled: Unimplemented WebDriver command
+  [test_no_browsing_context]
+    expected: ERROR
+
+  [test_format_and_dimensions]
```

Comment:
It would be good to understand what fails in this test.

Reply:
The test uses [`element_rect`](https://github.com/servo/servo/blob/master/tests/wpt/web-platform-tests/webdriver/tests/take_element_screenshot/__init__.py) which in turn uses `ExecuteScript` to get an object containing the element's dimensions. Since we can't have objects returned from `ExecuteScript` yet, the test fails.